src/main/java/dev/ebrydeu/spring_boot_library/domain/entities/User.java

```diff
@@ -0,0 +1,24 @@
+package dev.ebrydeu.spring_boot_library.domain.entities;
+
+import lombok.*;
+import jakarta.persistence.*;
+import java.util.UUID;
+@Data
```

Comment:
Remove it. Explanation on why [link](https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/#avoid-data)